protected/humhub/modules/admin/models/forms/PeopleSettingsForm.php

```diff
@@ -45,6 +46,7 @@ public function init()
         $this->detail3 = Yii::$app->settings->get('people.detail3', '');
         $this->defaultSorting = Yii::$app->settings->get('people.defaultSorting', 'lastlogin');
         $this->defaultSortingGroup = Yii::$app->settings->get('people.defaultSortingGroup', '');
+        $this->reduceFilterOptions = Yii::$app->settings->get('people.reduceFilterOptions', true);
```

Comment:
Thanks for the UI option, but a module option like this is probably better. The setting is rather very high level.
https://github.com/humhub/humhub/pull/6806/files/c1c027768714a5a03932a47ac72c34b3ea54997a..b328ec15951aaccb1d79488b65b6b8bdfdb20848